src/common/gamedata.rs

```diff
@@ -568,13 +569,184 @@ impl GameData {
     }
 
     /// Gets the order of the mount.
-    pub fn find_mount_order(&mut self, mount_id: u16) -> Option<i16> {
+    pub fn find_mount_order(&mut self, mount_id: u32) -> Option<i16> {
         let instance_content_sheet =
             MountSheet::read_from(&mut self.resource, Language::English).unwrap();
-        let mount_row = instance_content_sheet.get_row(mount_id as u32)?;
+        let mount_row = instance_content_sheet.get_row(mount_id)?;
 
         mount_row.Order().into_i16().copied()
     }
+
+    /// Gets the Item ID of the Orchestrion Roll.
+    pub fn find_orchestrion_item_id(&mut self, orchestrion_id: u32) -> Option<u32> {
+        let mut result = None;
+        'outer: for page in &self.item_pages {
+            for row in &page.rows {
+                let ExcelRowKind::SingleRow(single_row) = &row.kind else {
+                    panic!("Expected a single row!");
+                };
+
+                let physis::exd::ColumnData::UInt8(filter_group) = &single_row.columns[13] else {
```

Comment:
A lot of this (and other cases) and be replaced with the much shorter `into_X` helper funcs

Reply:
Done!